__tests__/Unit/hooks/tasksApi.test.tsx

```diff
@@ -30,11 +30,15 @@ function Wrapper({
 }: PropsWithChildren<Record<string, never>>): JSX.Element {
     return <Provider store={store()}>{children}</Provider>;
 }
-
 describe('useGetAllTasksQuery()', () => {
     test('returns all tasks', async () => {
+        const dev = false;
         const { result, waitForNextUpdate } = renderHook(
-            () => useGetAllTasksQuery(),
+            () =>
+                useGetAllTasksQuery({
+                    dev: dev as boolean,
+                    status: 'all',
+                }),
```

Comment:
Please keep the previous state as is.

Reply:
We have to give `dev` parameter as it is not optional.